rust-arroyo/src/processing/strategies/run_task.rs

```diff
@@ -69,7 +63,7 @@ impl<TPayload, TTransformed: Send + Sync> ProcessingStrategy<TPayload>
             return Err(SubmitError::MessageRejected(MessageRejected { message }));
         }
 
-        let next_message = (self.function)(message).map_err(SubmitError::InvalidMessage)?;
+        let next_message = (self.function)(message)?;
```

Comment:
does that mean that function provided by the user also needs to handle backpressure now?

Reply:
no, it means it can emit backpressure (that's the goal, to be able to emit backpressure from runtask, something you can already do in python